.travis.yml

```diff
@@ -30,69 +49,57 @@ matrix:
       env: TF_PACKAGE=tensorflow==1.1.0 KERAS_PACKAGE=keras==2.0.0 MPI=MPICH
     - python: "3.6"
       env: TF_PACKAGE=tensorflow==1.1.0 KERAS_PACKAGE=keras==2.0.0 MPI=MPICH
-    - python: "3.6"
+    - python: "3.4"
```

Comment:
why do we run tf-nightly with 3.4 now?

Reply:
This is exclusion matrix.  We used to run it with 3.4.  Now we run it with 3.6 since the base image is upgraded and it doesn't have 3.4 anymore.